integration/containerd/cri/integration-tests.sh

```diff
@@ -127,19 +127,20 @@ create_containerd_config() {
 
 cat << EOT | sudo tee "${CONTAINERD_CONFIG_FILE}"
 [plugins]
-  [plugins.cri]
-    [plugins.cri.containerd]
-        default_runtime_name = "$runtime"
-      [plugins.cri.containerd.runtimes.${runtime}]
-        runtime_type = "${runtime_type}"
-        $( [ $kata_annotations -eq 1 ] && \
-        echo 'pod_annotations = ["io.katacontainers.*"]' && \
-        echo '        container_annotations = ["io.katacontainers.*"]'
-        )
-        [plugins.cri.containerd.runtimes.${runtime}.options]
-          Runtime = "${containerd_runtime}"
-[plugins.linux]
-       shim = "${containerd_shim_path}"
+  [plugins."io.containerd.grpc.v1.cri"]
+    [plugins."io.containerd.grpc.v1.cri".containerd]
+      default_runtime_name = "$runtime"
+      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.${runtime}]
+          runtime_type = "${runtime_type}"
+          $( [ $kata_annotations -eq 1 ] && \
+          echo 'pod_annotations = ["io.katacontainers.*"]' && \
+          echo -n '          container_annotations = ["io.katacontainers.*"]'
+          )
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.${runtime}.options]
+            BinaryName = "${containerd_runtime}"
+[plugins."io.containerd.runtime.v1.linux"]
```

Comment:
Should add some spaces to align to `[plugins."io.containerd.grpc.v1.cri"]`?

Reply:
Yes, it should, thanks :smile: